pysam/libcalignedsegment.pyx

```diff
@@ -934,15 +927,16 @@ cdef class AlignedSegment:
             This field contains the index of the reference sequence in
             the sequence dictionary. To obtain the name of the
             reference sequence, use
-            :meth:`pysam.AlignmentFile.getrname()`
+            :meth:`get_reference_name()`
 
         """
         def __get__(self): return self._delegate.core.tid
         def __set__(self, tid): self._delegate.core.tid = tid
 
     property reference_start:
```

Comment:
might misunderstand, but reference_name property does already exist in AlignedSegment